woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php

```diff
@@ -121,6 +121,15 @@ public function setup_configs() {
 			$this->is_sandbox_mode = $this->configs->settings['sandbox_mode_enabled'] === 'yes';
 			$this->private_key = $this->is_sandbox_mode ? $this->configs->settings['sandbox_private_key'] : $this->configs->settings['live_private_key'];
 			$this->public_key = $this->is_sandbox_mode ? $this->configs->settings['sandbox_public_key'] : $this->configs->settings['live_public_key'];
+
+			if ($this->is_sandbox_mode) {
+				$warning_message = "You are currently in Sandbox mode, in this mode, none of your transactions will be processed.";
```

Comment:
When there messages like that, we always use the function `__` and the id `'woocommerce-gateway-ebanx'`. In this case the variable will be: `$warning_message = __('You are currently in Sandbox mode, in this mode, none of your transactions will be processed.', 'woocommerce-gateway-ebanx');`.

Another thing, always use simple quotes :)